src/v/archival/ntp_archiver_service.cc

```diff
@@ -803,8 +792,28 @@ ss::future<cloud_storage::upload_result> ntp_archiver::upload_manifest(
       "Uploading manifest, path: {}",
       manifest().get_manifest_path());
     auto units = co_await _parent.archival_meta_stm()->acquire_manifest_lock();
-    co_return co_await _remote.upload_manifest(
+
+    auto upload_insync_offset
+      = _parent.archival_meta_stm()->get_insync_offset();
+    vlog(
```

Comment:
nit: with this log message here, shouldn't we remove the identical one in maybe_upload_manifest?